cmd/api/routers/routers.go

```diff
@@ -0,0 +1,32 @@
+// Copyright KubeArchive Authors
+// SPDX-License-Identifier: Apache-2.0
+
+package routers
+
+import (
+	"fmt"
+	"github.com/gin-gonic/gin"
+	"net/http"
+)
+
+// FIXME This will be taken from a shared pkg with sink based on the DB schema
+// Just for the first approach
+type resources struct {
+	Kind       string   `json:"kind"`
+	APIVersion string   `json:"apiVersion"`
+	Items      []string `json:"items"`
+}
+
+// GetAllResources responds with the list of resources of a specific type across all namespaces
+func GetAllResources(c *gin.Context) {
+	group := c.Param("group")
+	version := c.Param("version")
+	pluralResourceType := c.Param("resourceType")
+	response := resources{
+		// FIXME The plural of the resource type can be different of adding an `s` to the kind
+		Kind:       pluralResourceType[:len(pluralResourceType)-1],
```

Comment:
There is a go package you can use to convert a word from plural to singular form (instead of just trimming the last character):

https://github.com/gertd/go-pluralize

Reply:
I didn't want to use an external library for this. The resources can be made up names with a non-predictable singular/plural. 

As this is a mock response for now (we don't query the DB yet) I think we can go with whatever the name of the resource is, plural or singular, and we can face this in the future if we need to.